### ISSUE TITLE
Documentation issues - missing TenantID and Mail.Read scope

### ISSUE THREAD

**Issue by ATJSM** [microsoftgraph/msgraph-sdk-powershell#16]
Hi, I was testing the steps mentioned [here](https://github.com/microsoftgraph/msgraph-sdk-powershell). I was getting several errors and was able to fix them. Reporting them here in case documentation needs to be updated.

**Issue 1:**
`Connect-Graph -ClientId 'ID' -Scopes 'user.read, Calendars.Read, Files.Read'`. Based on my tests, I think the TenantID is also needed otherwise, I receive an error.

![image](https://user-images.githubusercontent.com/13952319/64289694-8b513d80-cf19-11e9-81a5-042fc5ad15f7.png)

With TenantID, it succeeded.

**Issue 2:**
`Get-MeMessage -Top 10 -Skip 10 -Select 'Id, Subject, CreatedDateTime' | Format-Table CreatedDateTime, Subject, Id`. This example returns your messages, but for this to work, we need to add 'Mail.Read' to the scopes in the previous step. Otherwise, you will get an 'Access Denied' error.

![image](https://user-images.githubusercontent.com/13952319/64289805-bcca0900-cf19-11e9-9271-7265f576f966.png)

So, to sum up, I needed to run this:
`Connect-Graph -ClientId 'ID' -Scopes 'user.read, Calendars.Read, Files.Read, Mail.Read' -TenantID 'tenantID'` instead of `Connect-Graph -ClientId 'ID' -Scopes 'user.read, Calendars.Read, Files.Read'`

**Comment by darrelmiller** [comment]
Issue1:  When you register your app, select the option that allows it to work on any tenant and then it will use the /common endpoint and you don't have to specify the tenant.   I realize that sounds backward, but that's the way it works.

Issue 2:  Sorry for the confusion, yes, you are correct.  You need Mail.Read to access messages.
